src/controllers/userProfileController.js

```diff
@@ -686,7 +686,9 @@ const userProfileController = function (UserProfile, Project) {
       }
       if (
         req.body.infringements !== undefined &&
-        (await hasPermission(req.body.requestor, 'infringementAuthorizer'))
+        ((await hasPermission(req.body.requestor, 'addInfringements')) ||
+          (await hasPermission(req.body.requestor, 'deleteInfringements')) ||
+          (await hasPermission(req.body.requestor, 'editInfringements')))
```

Comment:
```suggestion
```
With the added APIs for adding/updating/deleting infringements, they don't also need to be edited here. (e.g. 'addInfringements' would still allow you to delete using this API). 
Removing this if statement should make everything good I think.

Reply:
I've removed if statement.